x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -154,6 +155,11 @@
     public static final NamedClusterPrivilege CANCEL_TASK = new ActionClusterPrivilege("cancel_task",
         Set.of("cluster:admin/tasks/cancel"));
 
+    public static final NamedClusterPrivilege MANAGE_ENROLLMENT = new ActionClusterPrivilege("manage_enrollment",
+        Set.of("cluster:admin/xpack/enrollment/*"));
+    public static final NamedClusterPrivilege ENROLL = new ActionClusterPrivilege("enroll",
+        Set.of("cluster:admin/xpack/enrollment/enroll"));
```

Comment:
Let's make this `cluster:admin/xpack/enrollment/enroll*` so that we can have `cluster:admin/xpack/enrollment/enroll_client` and `cluster:admin/xpack/enrollment/enroll_node` in the other APIs